vyper/functions/functions.py

```diff
@@ -134,7 +134,7 @@ def _convert(expr, context):
     #     assert typ in ('int128', 'uint256', 'address', 'bytes')
```

Comment:
Why is this stuff commented out?

Reply:
Remnants of the `convert` PR a while a go. Will remove it in a separate PR.

Reply:
Created https://github.com/ethereum/vyper/pull/744.